src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.cs

```diff
@@ -162,12 +177,16 @@ private bool StartNewModelComputation(ICompletionService completionService, Comp
 
             this.sessionOpt = new Session(this, computation, GetCompletionRules(), Presenter.CreateSession(TextView, SubjectBuffer, null));
 
-            var completionProviders = triggerInfo.TriggerReason == CompletionTriggerReason.Snippets
+            var completionProviders = trigger is DisplaySnippetsCompletionTrigger
```

Comment:
should we just use a well known name for this case?  this seems pretty hacky having to know exactly the class that was instantiated.